proto/decentraland/kernel/apis/testing.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package decentraland.kernel.apis;
+
+service PlayersService {
+  // sends a test result to the test runner
+  rpc LogTestResult(TestResult) returns (TestResultResponse) {}
+  // send a list of all planned tests to the test runner
+  rpc Plan(TestPlan) returns (TestPlanResponse) {}
+  // sets the camera position in the engine
+  rpc SetCameraPosition(SetCameraPositionTestCommand) returns (SetCameraPositionTestCommandResponse) {}
+}
+
+message Vector3 {
```

Comment:
can we reuse the common.Vector3 ?

Reply:
not for apis, those need to be self-contained